test/service/test_spreadsheet_upload_service.py

```diff
@@ -20,38 +20,24 @@ def setUp(self) -> None:
 
     def test_upload_success(self):
         # given:
-        self.importer.create_update_spreadsheet = Mock()
+        self.importer.update_spreadsheet_with_uuids = Mock()
```

Comment:
Some of the setup code is copied in a few tests, I would refactor and create a reusable function.

Reply:
I've moved them all up to the setUp function.